regression/cbmc/json-interface1/test_wrong_flag.desc

```diff
@@ -0,0 +1,6 @@
+CORE broken-smt-backend
```

Comment:
:grey_question: What is the `broken-smt-backend` for?

Reply:
The extra --cprover-smt2 flag would be ignored.